packages/core/src/components/slider/slider.tsx

```diff
@@ -53,12 +53,13 @@ export class Slider extends AbstractPureComponent2<ISliderProps> {
 
     public render() {
         const { initialValue, value, onChange, onRelease, ...props } = this.props;
+        const intent = props.intent || Intent.PRIMARY;
```

Comment:
same here, please use `public static defaultProps`

Reply:
this change has also now been made